packages/@react-spectrum/table/docs/Table.mdx

```diff
@@ -0,0 +1,783 @@
+<PokemonTable selectionMode="multiple" />
+```
+
+Table also supports a `disallowEmptySelection` prop which forces the user to have at least one row in the Table selected at all times.
+
+```tsx example
+// Using the same table as above
+<PokemonTable selectionMode="single" disallowEmptySelection />
+```
+
+### Sorting
+Table supports sorting its data when a column header is pressed. To designate that a Column should support sorting, provide it with
+the `allowsSorting` prop. The Table accepts a `SortDescriptor` prop that defines the current column key to sort by and the sort direction (ascending/descending).
+When the user presses a column's sort icon, the column's key and sort direction is passed into the `onSortChange` callback, allowing you to update
+the `SortDescriptor` appropriately.
+
+This example performs client side sorting by passing a `sort` function to the [useAsyncList](../react-stately/useAsyncList.html) hook.
+See the docs for more information.
+
+```tsx example
+function AsyncSortTable() {
+  let columns = [
+    {name: 'Name', key: 'name'},
+    {name: 'Height', key: 'height'},
+    {name: 'Mass', key: 'mass'},
+    {name: 'Birth Year', key: 'birth_year'}
+  ];
+
+  let list = useAsyncList({
+    getKey: (item) => item.name,
+    async load({signal, cursor}) {
+      if (cursor) {
+        cursor = cursor.replace(/^http:\/\//i, 'https://');
+      }
+
+      let res = await fetch(cursor || `https://swapi.dev/api/people/?search`, {signal});
+      let json = await res.json();
+
+      return {
+        items: json.results,
+        cursor: json.next
+      };
+    },
+    async sort({items, sortDescriptor}) {
+      return {
+        items: items.slice().sort((a, b) => {
+          let cmp = a[sortDescriptor.column] < b[sortDescriptor.column] ? -1 : 1;
+          if (sortDescriptor.direction === 'descending') {
+            cmp *= -1;
+          }
+          return cmp;
+        })
+      };
+    }
+  });
+
+  return (
+    <Flex height="size-2000" flexGrow={1} maxWidth="size-6000">
+      <Table aria-label="Example table with client side sorting" height="100%" width="100%" sortDescriptor={list.sortDescriptor} onSortChange={list.sort}>
+        <TableHeader columns={columns}>
+          {(column) => (
+            <Column allowsSorting align={column.key !== 'name' ? 'end' : 'start'}>
+              {column.name}
+            </Column>
+          )}
+        </TableHeader>
+        <TableBody
+          items={list.items}
+          isLoading={list.isLoading}
+          onLoadMore={list.loadMore}>
+          {(item) => (
+            <Row key={item.name}>{(key) => <Cell>{item[key]}</Cell>}</Row>
+          )}
+        </TableBody>
+      </Table>
+    </Flex>
+  );
+}
+```
+
+## Props
+
+### Table props
+
+<PropTable component={{
+  props: {
+    properties: Object.fromEntries(Object.entries(tableTypes.exports.SpectrumTableProps.properties))
+  }
+}} links={tableTypes.links} />
+
+### Table header props
```

Comment:
Open to opinions regarding all the extra prop tables, felt like it was worth including since it didn't exactly follow the collection api

Reply:
Seems like a good thing to me
We don't have to do this right now necessarily, but i wonder if we should put some sort of thing Divider between the tables, they kinda run together right now

Reply:
I'll see what I can do

Reply:
I can add `hr`s that look like this:
![image](https://user-images.githubusercontent.com/9661127/115468487-026b4080-a1e8-11eb-8cc8-8f1419cab7ec.png)

What do you think? Perhaps they should be thicker?

Reply:
![image](https://user-images.githubusercontent.com/9661127/115470118-ace46300-a1ea-11eb-965a-bcb3f429af52.png)
Via dividers

Reply:
I think the thin (S) Divider is better? otherwise it looks like we're separating sections?

Reply:
I tried that, on 2nd thought the divider kinda looks weird... I modified PropTable to accept a style prop instead so I could add marginBottom